text/0000-add-ember-page-title-addon.md

```diff
@@ -0,0 +1,48 @@
+- Start Date: 2020-06-24
+- Relevant Team(s): Ember CLI
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+- Authors: Benjamin Jegard, Melanie Sumner, Ricardo Mendes
+
+# Add ember-page-title to the app blueprint
+
+## Summary
+
+Add [`ember-page-title`](https://github.com/adopted-ember-addons/ember-page-title) to the default blueprint for new Ember apps as a way to provide improved out-of-the-box (OOB) accessibility for Ember applications.
+
+## Motivation
+
+This RFC is part of the work made by the Ember.js Accessibility Strike Team to ensure that newly created ember apps have no accessibility issues.
+
+Users with assistive technology rely on the page title to know if they are on the correct page of a website.
+Adding this addon will provide developers a simple solution to achieve the [WCAG Success Criterion 2.4.2: Page Titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
+
+## Detailed design
+
+1. Make [`ember-page-title`](https://github.com/adopted-ember-addons/ember-page-title) an official Ember addon by transferring `ember-page-title` repo to the [Ember CLI org](https://github.com/ember-cli) (it's currently in the [Adopted Ember Addons org](https://github.com/adopted-ember-addons))
+2. Add the dependency to the app blueprint here: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json#L19
+3. Update app blueprint to include `<HeadLayout />` at the top of [application.hbs](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/templates/application.hbs) as well as a helpful comment explaining what it is used for
```

Comment:
This will not be required after this gets done - https://github.com/adopted-ember-addons/ember-page-title/issues/167 - already in progress.

Reply:
🔝  is done but there are 2 blueprint update issues still to do:

* https://github.com/adopted-ember-addons/ember-page-title/issues/174
* https://github.com/ember-fastboot/ember-cli-fastboot/issues/792

Reply:
Great that's good news.
I'll wait until the new version is released to remove this.

Reply:
ember-page-title v6 has been released, so we can remove these lines 👍